weaver/sdks/besu/node/package.json

```diff
@@ -17,7 +17,7 @@
     "@hyperledger/cacti-weaver-protos-js": "2.0.0-alpha.2",
     "log4js": "6.9.1",
     "web3": "1.10.0",
-    "web3-utils": "1.10.0"
+    "web3-utils": "1.7.0"
```

Comment:
This is a downgrade. @petermetz Do you know how to avoid this from dependabot?

Reply:
@sandeepnRES Ouch, thank for catching that! Not sure how to prevent it unfortunately. It also erodes my trust in the robots quite a bit because this is not the first time this has happened so it seems to be a recurring bug now instead of just a weird one-off. Makes me consider if we should switch to a different tool altogether.
We definitely need something to perform the role of checking for vulnerable dependencies, but if the automation is sending pull requests that actually introduce the vulnerabilities (with the downgrades) then it's worse than not having it at all.